addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -724,11 +724,14 @@ blockquote {
     // Those color classes color multiple elements when applied on a snippet.
     // Those rules are not important so that they can be overridden through
     // bg and text utility classes. **
-    .o_cc#{$index} {
+    .o_cc#{$index}, .o_cc .o_cc#{$index} {
```

Comment:
That cannot be right and especially not without comments. Always imagine being someone reaching this line, how could he ever understand why that would be right? Especially in combination with `$o-color-extras-nesting-selector`... there must be something wrong :grimacing: 

I actually have a hard time understaning it with the commit itself. Is that text-solid class really needed etc? How would that go with already colored text in databases? If that is actually needed, I wonder if it will be a good idea to keep this (even if your code seems to work great). Adding gradients on the default presets background is something really cool to me (most design SRI does uses small gradients as backgrounds and it is always better and make sense) but default gradient on text? Not so sure. Adding gradient on text yes... but by default? Really not sure.